src/main/resources/com/google/api/codegen/ruby/message.snip

```diff
@@ -1,32 +1,42 @@
 @extends "ruby/common.snip"
 
-@snippet generateFilename(file)
-  doc/{@context.getBasename(file)}.rb
-@end
-
-@snippet generateClass(file)
-  @let body = documentChildren(file)
-    {@licenseSection()}
+@snippet generate(doc)
+  @let body = documentChildren(doc)
+    {@licenseSection(doc.fileHeader)}
 
-    {@module(context.getGrpcModules(file).iterator, body) }
+    {@module(doc.fileHeader.modules.iterator, body) }
   @end
 
 @end
 
-@private documentMessages(containerElement)
-  @join element : context.filterDocumentingMessages(containerElement) on BREAK.add(BREAK)
-    @let document = context.methodDocComment(element)
-      @if document
-        {@toComments(document)}
+@private documentMessage(doc)
+  @if doc.lines
```

Comment:
Seems like there is shared code between this if and the else. Is that because you are trying to avoid an extra line?